Automattic/wp-calypso
# Proper naming for SVG properties in Example Domain Suggestions component

The `fill-rule` property for the SVG wasn't using the `camelCase` notation and React was complaining about it. Updated it to use the proper notation. 

No visual differences observed.

To test:

1. Checkout branch or use the Calypso.live link
2. Start signup
3. Reach the domains step. 
4. Make sure nothing is broken in the example domain graphic: 
![screen shot 2016-10-06 at 13 10 24](https://cloud.githubusercontent.com/assets/184938/19148621/4867ad90-8bc6-11e6-90fb-73d10e9476f3.png)
